packages/relay/src/lib/services/hapiService/hapiService.ts

```diff
@@ -289,6 +290,9 @@ export default class HAPIService {
     this.clientMain = this.initClient(this.logger, this.hederaNetwork);
     this.client = this.initSDKClient(this.logger);
     this.resetCounters();
+    if (this.clientMain.operatorAccountId) {
+      this.hbarLimitService.setOperatorAddress(this.clientMain.operatorAccountId.toSolidityAddress());
+    }
```

Comment:
same question as above, would this be considered technical debt that will be removed (if the logic is done in the service class) later when the draft PR is ready?